requirements.txt

```diff
@@ -40,4 +40,5 @@ torch==1.7.0
 
 # Beam nuggets for postgres
 beam-nuggets==0.17.0
-pg8000==1.16.5
\ No newline at end of file
+pg8000==1.16.5
+sqlalchemy==1.3.22  # pinning die to `ImportError: cannot import name '_literal_as_text' from 'sqlalchemy.sql.expression' with `from beam_nuggets.io import relational_db`
```

Comment:
I think there is a typo here: "die" -> "due"

Reply:
Sure! Resolved